### PR TITLE
Add SWPR token transfer permissions on localhost for swapr guild

### PR DESCRIPTION
**Pull request by MiltonTulli**
# Description

Allow Swpr guild to transfer swpr token.

Closes https://github.com/DXgovernance/DAVI/issues/166

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [x] My code follows the existing style of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any UI changes have been tested and made responsive for mobile views

### CODE REVIEW

File: scripts/dev.ts [hunk 1]
```diff
@@ -836,6 +841,71 @@ async function main() {
...
+      data: {
+        guildName: 'SWPRGuild',
+        to: ['PermissionRegistry'],
+        callData: [
+          new web3.eth.Contract(PermissionRegistry.abi).methods
+            .setETHPermission(
+              networkContracts.addresses.SWPRGuild,
+              networkContracts.addresses.SWPR,
+              ERC20_TRANSFER_SIGNATURE,
+              web3.utils.toWei('0').toString(),
```
Comment (rossneilson) [inline scripts/dev.ts:876]: I think this shouldn't be a string

Comment (MiltonTulli) [reply scripts/dev.ts]: It's confusing. Some fn calls are using string.
Removed!

Comment (rossneilson) [reply scripts/dev.ts]: Hmm, perhaps it is handled fine here

**Review by dcrescimbeni — APPROVED** [review]

**Comment by Filipv95** [comment]
@MiltonTulli basically I can transfer tokens without setting permission to SWPR token first is that correct? Is there somethingelse that should be checked in this PR?

**Review by Kenny-Gin1 — APPROVED** [review]
LGTM ⭐

**Comment by MiltonTulli** [comment]
> @MiltonTulli basically I can transfer tokens without setting permission to SWPR token first is that correct? Is there somethingelse that should be checked in this PR?

Hey @Filipv95! Yes, in the issue (https://github.com/DXgovernance/DAVI/issues/166) you mentioned that you wanted to execute a proposal with a transfer action of 3 SWPR tokens from swpr guild but it was failing due to permissions. If you manually first create a proposal to allow that (setting permissions to execute transfers from the guild to the token address) and vote/execute and then you create the same proposal that you described on the issue it should work ok. What I did in this pr is saving you the 1st step by creating and executing the permission proposal vía script.

**Comment by Filipv95** [comment]
@MiltonTulli Understood, thank you. Everything works as expected!

**Pull request merged**